src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -208,13 +215,17 @@ private void parseCvrFile(
             continue;
           }
         }
-        // validate precinct
+        // validate precinct (may not exist for older data sets)
         Integer precinctId = (Integer) adjudicatedData.get("PrecinctId");
-        if (precinctId != null && !this.precincts.containsKey(precinctId)) {
+        if (precinctId != null && (this.precincts == null || !this.precincts
+            .containsKey(precinctId))) {
           Logger.severe("Precinct ID \"%d\" from CVR not found in manifest data!", precinctId);
           throw new CvrParseException();
         }
-        String precinct = this.precincts.get(precinctId);
+        String precinct = null;
+        if (this.precincts != null) {
+          precinct = this.precincts.get(precinctId);
+        }
```

Comment:
Nit: you could simplify this a bit:
`String precinct = this.precincts != null ? this.precincts.get(precinctId) : null;`